Sources/FluentKit/Model/AnyModel.swift

```diff
@@ -34,10 +34,15 @@ extension AnyModel {
     }
 
     var anyID: AnyID {
-        guard let id = Mirror(reflecting: self).descendant("_id") as? AnyID else {
-            fatalError("id property must be declared using @ID")
+        for (nameC, child) in _FastChildSequence(subject: self) {
```

Comment:
Can we get some docs? 😇

Reply:
There's a few constants I'd love to understand

Reply:
Okay, yeah, that's fair, I should've at least added the comments for the ASCII codes 😅